libraries/botbuilder/src/botFrameworkAdapter.ts

```diff
@@ -8,9 +8,10 @@
 
 import { STATUS_CODES } from 'http';
 import { arch, release, type } from 'os';
+import { HttpClient } from '@azure/ms-rest-js';
```

Comment:
Is this the interface or an abstract class?

Reply:
It is just the interface. I wondered if i should leave it out.?

Reply:
If it’s only used in the private method as a parameter I would leave it out, change the type to any and perhaps leave a comment.

Reply:
removed